lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -189,6 +189,15 @@ def verify_credentials
 
       private
 
+      def supports_network_tokenization_brands
+        result = Braintree::MerchantGateway.new(@braintree_gateway).provision_raw_apple_pay
```

Comment:
This method seems unnecessary, as we can now have only:
```
def supports_network_tokenization?
  response = Braintree::MerchantGateway.new(@braintree_gateway).provision_raw_apple_pay
  response.success? && response.supported_networks.present?
end
```

This is if we decide to address the concerns on `provision_raw_apple_pay` though.